src/SFA.DAS.Location.Application.UnitTests/Search/Queries/WhenHandlingTheSearchLocationRequest.cs

```diff
@@ -1,33 +1,35 @@
 using System.Collections.Generic;
+using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
 using AutoFixture.NUnit3;
 using FluentAssertions;
 using Moq;
 using NUnit.Framework;
-using SFA.DAS.Location.Application.Location.Queries.SearchLocations;
+using SFA.DAS.Location.Application.Search.Queries.SearchLocations;
 using SFA.DAS.Location.Domain.Interfaces;
+using SFA.DAS.Location.Domain.Models;
 using SFA.DAS.Testing.AutoFixture;
 
-namespace SFA.DAS.Location.Application.UnitTests.Location.Queries
+namespace SFA.DAS.Location.Application.UnitTests.Search.Queries
 {
     public class WhenHandlingTheSearchLocationRequest
     {
-        [Test, MoqAutoData]
+        [Test, MoqInlineAutoData("Man")]
```

Comment:
Is this test case necessary? Looking at the test, it looks like it _should_ work without setting the search term